adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -955,6 +961,51 @@ func createRoute(params *routeCreateParams) *routev3.Route {
 			}
 			action.Route.RateLimits = []*routev3.RateLimit{&rateLimit}
 		}
+
+		if config.Envoy.RateLimit.Enabled {
```

Comment:
Check if we can add it at the vhost level, so that the config_dump remains small

Reply:
Vhost level does not work